tests/test_dse.py

```diff
@@ -560,10 +569,14 @@ def test_uncontracted_shape(self, rotate):
                                               'cire-rotate': rotate}))
 
         # Check code generation
+        trees = retrieve_iteration_tree(op1)
```

Comment:
here's another (alternative) idea: add, to `common.py`, a utility function such as `get_blocked_nests`, so that you can then do:

```
trees = get_blocked_nests(op1)
assert len(trees) == 2  # number of expected blocked nests
...
```